Sources/PackageGraph/PackageGraph.swift

```diff
@@ -24,10 +24,10 @@ public struct PackageGraph {
 
     // FIXME: These are temporary.
     public let modules: [Module]
-    public let externalModules: Set<Module>
+    public let externalModules: [Module]
```

Comment:
I could keep it as a set and then sort the modules alphabetically when I use it to add the modules to the pbxproj. Personally I would really like them to be sorted anyway because it makes it much easier to quickly find a package.